dashboard/test/integration/db_query_test.rb

```diff
@@ -139,6 +139,10 @@ def setup
       published_state: SharedCourseConstants::PUBLISHED_STATE.stable
     )
     CourseOffering.add_course_offering(unit)
+
+    # make sure the new unit is in the cache
+    setup_script_cache
```

Comment:
this test was a bit of a mystery. the change which caused this test to break was 30c89f3adc2949ed943a618b23bbe5b3641cabaa. I was not able to determine why, but when I went to debug it looked like the `@script` object under test was in an inconsistent state. this was because the script cache is enabled but the script was not in the cache. so, I solved the problem this way to make sure that the newly created script object was in the script cache.